.github/workflows/ideaspace-staging.yaml

```diff
@@ -0,0 +1,54 @@
+name: staging-image
+on:
+  push:
+    branches:
+      - staging/ideaspace
+    paths-ignore:
+      # Ignore when changes is made by flux
+      - 'staging/**'
+      - 'production/**'
+      # Ignore markdown files
+      - '**/*.md'
+    # Allows you to run this workflow manually from the Actions tab
+    workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      # https://stackoverflow.com/questions/60942067/get-current-date-and-time-in-github-workflows
+      - name: Get current date
+        id: date
+        run: echo "date=$(date +'%Y%m%d%H%M')" >> $GITHUB_ENV
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: devlaunchers/platform-website
+          tags: |
+            type=raw,value=staging-{{sha}}-${{ steps.date.outputs.date }}
```

Comment:
Let's prefix the tag with `staging-ideaspace`.

Reply:
@oriyomibadmus can we update this?